SofaKernel/modules/Sofa.Type/Sofa.Type_test/Quater_test.cpp

```diff
@@ -340,8 +342,8 @@ TEST(QuaterTest, QuaterdOperatorDivRealSelf)
 //
 //TEST(QuaterTest, QuaterdVecQuatMult)
 //{
-//    Quater<double> quat, quatMult;
-//    sofa::defaulttype::Vec<3, double> vec(1, 2, 3);
+//    Quat<double> quat, quatMult;
+//    sofa::type::Vec<3, double> vec(1, 2, 3);
```

Comment:
Should we keep this commented code?

Reply:
obviously not but I wont clean the files 😴 in this PR